libs/features/mas/web-components/test/merch-card.mini-compare.test.html.js

```diff
@@ -52,7 +52,7 @@ runTests(async () => {
                         const el =
                             miniCompareChart.shadowRoot.querySelector(selector);
                         if (!el) return 0;
-                        return parseFloat(window.getComputedStyle(el).height);
+                        return parseInt(window.getComputedStyle(el).height);
```

Comment:
Providing a radix might be even safer
```suggestion
                        return parseInt(window.getComputedStyle(el).height, 10);
```